client/src/elm/Pages/Prenatal/Activity/Utils.elm

```diff
@@ -3593,9 +3593,16 @@ resolveMedicationTreatmentFormInputsAndTasksCommon language currentDate setBoolI
 
                     else
                         ( [], [] )
+
+                stillTaking =
+                    if config.latestMedicationTreatment == latestMedicationTreatmentForHIV assembled then
```

Comment:
1. Why this condition? Check `task`.
2. This may not be very significant duplication, but you can avoid having `viewQuestionLabel language` twice.

Reply:
@anvmn About this condition, Based on the task, that question on **ARV's** will be displayed on HIV Medication and on others it will be the general message.

Reply:
@Diama1 why not asking about task directly?

Reply:
@Diama1 

Isn't this simpler:

```
if task == TreatmentReviewHIV then 
  ... 
else 
...
```